variables.tf

```diff
@@ -133,10 +133,21 @@ variable "enable_certmanager" {
 variable "fluentbit_group" {
   description = "fluent bit cloudwatch log group"
   default = "/aws/eks/fluentbit-cloudwatch/jupyter/logs"
-  
+
 }
 
 variable "fluentbit_stream_name" {
   description = "fluent bit cloudwatch log stream"
-  default = "fluentbit-jupyter"  
-}
\ No newline at end of file
+  default = "fluentbit-jupyter"
+}
+
+variable "sns_name" {
+  description = "name of the sns topic"
+  default = "jupyter-alerts"
+  
+}
+
+variable "webhook_url" {
+  description = "webook url of slack"
+  
```

Comment:
could you add a default value here? it can be something like "SLACK_URL". And in the lamda function, you can check the URL. If it is not valid, exit.